### DISCUSSION TITLE
budibase app for mobile

### DISCUSSION THREAD

**Discussion by KiriakiK** [Budibase/budibase#6579] [Help] [answered]
Hi, I'm new to budibase and I don't know a lot about it. I use it to build an app. Is there a way to upload this application that I am creating to Play Store for android devices? Or is the app I'm creating just a web app?

**Comment by gerhardmpl** [comment, marked as answer]
@KiriakiK it is a web app that currently needs the (self hosted or cloud) Budibase toolstack to run.

**Reply by KiriakiK** [reply]
Thanks for your reply.
But is there any way to make it a progressive web app so that users can also download it on their mobile or not?

**Reply by gerhardmpl** [reply]
@KiriakiK It is probably better for a Maintainer to answer this question, because I honestly do not know. But #1084 does not sound like it is possible yet.
